contracts/src/v0.8/automation/dev/test/AutomationRegistry2_3.t.sol

```diff
@@ -1611,6 +1611,43 @@ contract BillingOverrides is SetUp {
 }
 
 contract Transmit is SetUp {
+  function test_transmitRevertWithExtraBytes() external {
+    bytes32[3] memory exampleReportContext = [
+      bytes32(0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef),
+      bytes32(0xabcdef1234567890abcdef1234567890abcdef1234567890abcdef1234567890),
+      bytes32(0x7890abcdef1234567890abcdef1234567890abcdef1234567890abcdef123456)
```

Comment:
total nitpick, but it might be a little cleaner if we just use the `_random()` function to generate these bytes ex...
```sol
bytes32[3] memory exampleReportContext = [_random(), _random(), _random()];
```

Reply:
Good call! I will take a note for this. will leave it as is for now since dont want tobother to get approvals again.